src/reanimated2/animation/decay.ts

```diff
@@ -38,6 +38,7 @@ export interface InnerDecayAnimation
   current: number;
 }
 
+const EPSILON = 0.01;
```

Comment:
Isn't it too much?

Reply:
@Latropos I know that you discussed this with Tomek offline, but let's write the conclusion in a comment to keep the informations in one place, please 🙏

Reply:
Sure! We want to have similar precision for decay and withSpring. Also we have tested, that it works well with `const EPSILON = 0.01;` and that we animate static spring for long time if epsilon is smaller.